src/umpire/strategy/DynamicPoolList.cpp

```diff
@@ -55,6 +55,16 @@ void DynamicPoolList::release()
   dpa.release();
 }
 
+std::size_t DynamicPoolList::numReleasableBlocks() const noexcept
```

Comment:
These methods should be prefixed with `get`.

Reply:
done.